### PR TITLE
UpdateUserFrameworkTest: added id field in framework for all request …

### PR DESCRIPTION
**Pull request by iostream04**
…object

### CODE REVIEW

File: sunbird_service_api_test/src/test/java/org/sunbird/integration/test/user/UpdateUserFrameworkTest.java [hunk 1]
```diff
@@ -157,5 +157,6 @@ private void beforeTest() {
     variable("gradeLevel", System.getenv("sunbird_user_framework_grade_level"));
     variable("medium", System.getenv("sunbird_user_framework_medium"));
     variable("subject", System.getenv("sunbird_user_framework_subject"));
+    variable("id", System.getenv("sunbird_user_framework_id"));
```
Comment (amolnin) [inline sunbird_service_api_test/src/test/java/org/sunbird/integration/test/user/UpdateUserFrameworkTest.java]: rename to framework-id

File: Jenkinsfile [hunk 2]
```diff
@@ -29,6 +29,7 @@ pipeline {
         sunbird_user_framework_subject = "English"
         content_reviewer_user="ft_reviewer2@org.com"
         content_reviewer_password="password"
+       	sunbird_user_framework_id="NCFCOPY"
```
Comment (amolnin) [inline Jenkinsfile]: formatting

File: sunbird_service_api_test/src/test/java/org/sunbird/integration/test/user/UpdateUserFrameworkTest.java (hunk 1 above)
Comment (amolnin) [reply sunbird_service_api_test/src/test/java/org/sunbird/integration/test/user/UpdateUserFrameworkTest.java]: add one more test-case - where frame-work id is missing

one more test-case - where framework id is invalid.

Comment (iostream04) [reply sunbird_service_api_test/src/test/java/org/sunbird/integration/test/user/UpdateUserFrameworkTest.java]: Done !

File: Jenkinsfile (hunk 2 above)
Comment (iostream04) [reply Jenkinsfile]: Done !

**Pull request merged**